templates/designatebackendbind9/config-named/options.conf

```diff
@@ -0,0 +1,38 @@
+options {
+        allow-new-zones yes;
+        directory "/var/named-persistent";
+
+        # The default for named is to listen on everything and it's controlled
+        # by different settings for different IP versions. If the objective is to
+        # restrict to listening on a given address, we need to explicitly turn
+        # listening *off* for the different IP version.
+
+        # TODO: The '*'s need to be replaced by actual addresses.
+{{ if eq .IPVersion "4" }}
+        listen-on port 53 { *; };
+        listen-on-v6 { none; };
+{{ else if eq .IPVersion "6" }}
+        listen-on-v6 port 53 { *; };
+        listen-on { none; };
+{{ end }}
+
+        {{/* Allowing on the network attachment CIDR should be sufficient accesss
+             control as it as the admin should only connect designate pods to
+             the designate network */}}
+        allow-notify { {{ .AllowCIDR }}; };
+
+        {{/* Extra bind customization is handled by passing values through the spec and is
+             generated in place here. This is necessary as apparently you cannot have
+             mulitple 'options' sections in bind9 configuration */}}
+{{  if .CustomBindOptions }}
+    {{ range $index, $val := .CustomBindOptions }}
+            {{ $val }}
+    {{ end }}
+{{ end }}
+
+        minimal-responses yes;
+        multi-master yes;
```

Comment:
I wonder why would we want this one to be "yes"? I don't see it on the designate repo, and the default value is "no"
https://bind9.readthedocs.io/en/latest/reference.html#namedconf-statement-multi-master

Reply:
Hrm. We'll need to check the tripleo install and the jira issue. That's the reference we should use.

Reply:
Yeah, multi-master is yes in tripleo. IIRC the "master's" in this case are the mdns servers, which there are many.